src/NServiceBus.Core/Transports/IDequeueMessages.cs

```diff
@@ -2,23 +2,21 @@
 {
     using System;
 
-    //public class Dequeue
-
     /// <summary>
     /// Interface to implement when developing custom dequeuing strategies.
     /// </summary>
-    public interface IDequeueMessages : IObservable<MessageAvailable>
+    public interface IDequeueMessages
```

Comment:
Didn't we agree to rename to something like IProcessMessages, IMessageProcessor, IMessagePump...?